.travis.yml

```diff
@@ -15,11 +15,13 @@ matrix:
       dist: bionic
 
 before_install:
-- git clone https://github.com/spideruci/projects4testing.git
 - git clone https://github.com/spideruci/primitive-hamcrest.git
 - cd primitive-hamcrest  
 - mvn install
 - cd ..
 
+install:
+- cp .travis.settings.xml $HOME/.m2/settings.xml
+
```

Comment:
this copies over the settings.xml that we want to use *only* on Travis, and remains transparent to the local dev environment (unless you actually run that `cp` command)